### PR TITLE
Static SDCA Multiclass, and Multiclass Evaluator

### PR DESCRIPTION
**Pull request by TomFinley**
Related to #632.

### CODE REVIEW

File: src/Microsoft.ML.Data/StaticPipe/TrainerEstimatorReconciler.cs [hunk 1]
```diff
@@ -336,5 +336,70 @@ private sealed class ImplBool : Scalar<bool>
                 public ImplBool(BinaryClassifierNoCalibration rec) : base(rec, rec._inputs) { }
             }
         }
+
+        /// <summary>
+        /// A reconciler capable of handling the most common cases for binary classification with calibrated outputs.
```
Comment (sfilipi) [inline src/Microsoft.ML.Data/StaticPipe/TrainerEstimatorReconciler.cs]: >binary [](start = 71, length = 6)

multi-class?

File: src/Microsoft.ML.Data/StaticPipe/TrainerEstimatorReconciler.cs [hunk 2]
```diff
@@ -336,5 +336,70 @@ private sealed class ImplBool : Scalar<bool>
...
+        public sealed class MultiClassifier<TKeyValue> : TrainerEstimatorReconciler
+        {
+            /// <summary>
+            /// The delegate to create the multiclass trainer instance.
+            /// </summary>
+            /// <param name="env">The environment with which to create the estimator.</param>
+            /// <param name="label">The label column name.</param>
+            /// <param name="features">The features column name.</param>
+            /// <param name="weights">The weights column name, or <c>null</c> if the reconciler was constructed with <c>null</c> weights.</param>
+            /// <returns>A binary classification trainer estimator.</returns>
```
Comment (sfilipi) [inline src/Microsoft.ML.Data/StaticPipe/TrainerEstimatorReconciler.cs]: >binary [](start = 27, length = 6)

multi-class?

File: src/Microsoft.ML.Data/StaticPipe/TrainerEstimatorReconciler.cs [hunk 3]
```diff
@@ -336,5 +336,70 @@ private sealed class ImplBool : Scalar<bool>
                 public ImplBool(BinaryClassifierNoCalibration rec) : base(rec, rec._inputs) { }
             }
         }
+
+        /// <summary>
+        /// A reconciler capable of handling the most common cases for binary classification with calibrated outputs.
+        /// </summary>
+        public sealed class MultiClassifier<TKeyValue> : TrainerEstimatorReconciler
```
Comment (sfilipi) [inline src/Microsoft.ML.Data/StaticPipe/TrainerEstimatorReconciler.cs]: >MultiClassifier [](start = 28, length = 15)

MultiClassClassifier?

**Comment by TomFinley** [comment]
Closing what was unique has been merged to #882

**Pull request closed without merging**